static/js/theme-map/Maps/Providers/Mapbox.js

```diff
@@ -101,6 +101,7 @@ class MapboxPin extends ProviderPin {
     this._pinEl.style.position = 'absolute';
     this._pinEl.style.top = '0';
     this._pinEl.style.left = '0';
+    this._pinEl.classList.add('Answers-mapPin');
```

Comment:
Do you all think it's okay to just update the pin CSS class for MapBox? It doesn't look like Google Maps supports custom CSS classes on its pins, and I don't think those pins are intended to be customizable with CSS either.

Reply:
is it possible to select this pin without adding a css class? if adding a css class makes it a lot simpler this lgtm!

Reply:
Is this test currently only taking screenshots with Mapbox maps? That's totally fine for now, but it's possible in the future we want Google Maps snapshots as well. Does Google have CSS classes on its pins that we can target?

Reply:
I'll investigate if more advanced query selectors can find the pin without this CSS class. And yes Tom, we only take screenshots with Mapbox maps. Google doesn't have CSS classes on its pins, however I will investigate if there is a more advanced selector that I could use to find one

Reply:
I've updated to use a query that no longer requires us to add a CSS class.  If we add Google maps support in the future, we can find a pin with the following: `document.querySelector('.gm-style img').parentElement`